content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,532 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /statistical-hypothesis-testing-python-implementation/
+title: Statistical Hypothesis Testing With Python Implementation
+description: This tutorial is a comprehensive explanation and implementation of statistical hypothesis testing methods such as T-Test, Chi-square test & ANOVA in Python.
+author: qoyum-olatunde-yusuf
+date: 2021-06-16T00:00:00-11:00
+topics: [Statistical Analysis, Hypothesis Testing, Data Science, Python ]
+excerpt_separator: <!--more-->
+images:
+ 
+ - url: /engineering-education/statistical-hypothesis-testing-python-implementation/hero.PNG
+alt: statistic hypothesis testing with python cover image
+---
+Hypothesis is the act of making observations and generating probing questions from the observations made in an attempt to come to a conclusion. Hypothesis testing is the test of assumptions on a population sample. It involves checking out whether a hypothesis should be accepted or not. In the business world and in real life, hypothesis testing has been of great importance in making worthy decisions off mere observations. Billions of data are being generated on a daily basis around the globe and this has made the relevance of hypothesis testing a stronghold in making business decisions.
+<!--more-->
+### Table of Contents
+1. [Prerequisites](#prerequisites)
+2. [Goals of the tutorial](#goals-of-the-tutorial)
+3. [Understanding hypothesis testing](#understanding-hypothesis-testing)
+4. [Confidence of null hypothesis](#confidence-of-null-hypothesis)
+5. [Statistical techniques for hypothesis testing](#statistical-techniques-for-hypothesis-testing)
+6. [Chi-square test - Python implementation](#chi-square-test-python-implementation)
+7. [T-Test - Python implementation](#t-test-python-implementation)
+8. [ANOVA - Python implementation](#anova-python-implementation)
+9. [Conclusion](#conclusion)
+10. [Additional resources]((#additional-resources))
```

Comment:
@wizardcalidad Like I mentioned, please remove the double brackets in the "Additional resources" tag
Also, sections 2,4,6,7,8 and 10 are not linking to the article at all. Please cross-check the headers and the title tags.